homeassistant/components/garages_amsterdam/coordinator.py

```diff
@@ -0,0 +1,34 @@
+"""Coordinator for the Garages Amsterdam integration."""
+
+from __future__ import annotations
+
+from odp_amsterdam import ODPAmsterdam, VehicleType
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN, LOGGER, SCAN_INTERVAL
+
+
+class GaragesAmsterdamDataUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class GaragesAmsterdamDataUpdateCoordinator(DataUpdateCoordinator[dict[str, garage]]):
```

Reply:
The function `all_garages` returns a list of garage models.